templatesOas/apis.mustache

```diff
@@ -44,12 +46,40 @@ export type {{operationIdCamelCase}}{{enumName}} =
 {{/stringEnums}}
 {{/isEnum}}
 {{/allParams}}
+{{/x-scapi-internal}}
```

Comment:
Can we include a mustache template in another mustache template. If that is possible may be we can avoid duplicate code